lms/djangoapps/commerce/api/v1/tests/test_views.py

```diff
@@ -32,21 +33,41 @@ def setUp(self):
                                                      currency=u'USD', sku=u'ABC123')
 
     @staticmethod
-    def _serialize_course_mode(course_mode):
+    def _serialize_course_mode(course_mode, verification_deadline=None):
         """ Serialize a CourseMode to a dict. """
         # encode the datetime (if nonempty) using DRF's encoder, simplifying
         # equality assertions.
         expires = course_mode.expiration_datetime
+        json_encoder = JSONEncoder()
         if expires is not None:
-            expires = JSONEncoder().default(expires)
+            expires = json_encoder.default(expires)
+
+        if verification_deadline is not None:
+            verification_deadline = json_encoder.default(verification_deadline)
+
         return {
             u'name': course_mode.mode_slug,
             u'currency': course_mode.currency.lower(),
             u'price': course_mode.min_price,
             u'sku': course_mode.sku,
             u'expires': expires,
+            u'verification_deadline': verification_deadline,
+        }
+
+    @classmethod
+    def _serialize_course(cls, course, modes=None, verification_deadline=None):
+        """ Serializes a course to a Python dict. """
+        data = {
+            u'id': unicode(course.id),
+            u'name': unicode(course.display_name),
+            u'modes': []
         }
 
+        modes = modes or []
+        data[u'modes'] = [cls._serialize_course_mode(mode, verification_deadline) for mode in modes]
```

Comment:
Updated test method. Also, update serializer to prefer the latest verification date when multiple values are provided.